src/reanimated2/Easing.js

```diff
@@ -252,3 +253,32 @@ export const Easing = {
   out,
   inOut,
 };
+
+function createChecker(worklet) {
+  function checkIfReaOne() {
+    'worklet'
+    if (arguments && (!_WORKLET)) {
+      for (let i = 0; i < arguments.length; i++) {
+        const arg = arguments[i];
+        if (arg && arg.__nodeID) {
+          throw new Error(
+            'Reanimated: Easing from V1 has been renamed to EasingNode.'
+          );
+        }
+      }
+    }
+    const res = worklet.apply(this, arguments);
+    if (res && (typeof res === 'function') && (!_WORKLET)) {
```

Comment:
if `res` is undefined or null it won't have typeof "function". But maybe we want to be explicit here.